library/lib.sh

```diff
@@ -227,6 +232,11 @@ rolesConvertToCollection() {
 --subrole-prefix $subrole_prefix"
 }
 
+rolesGetManagedNodes() {
+    local guests_yml=$1
```

Comment:
```suggestion
    local guests_yml="$1"
```

Reply:
I'm surprised that shellcheck didn't catch this.  Maybe the shellcheck rules have changed?  It used to be that shellcheck would flag any use of `var=$some_other_var` and would require `var="$some_other_var"` - maybe shellcheck is smart enough to know that `$1` in this case cannot be set by a "user"?

Reply:
Right, I noticed that this is one of few places where shellcheck does not require you to add quotes.